test_files/jsdoc/jsdoc.ts

```diff
@@ -128,3 +128,18 @@ Polymer({behaviors: ['test' as any]});
  * @template T2 Another user comment.
  */
 class Foo<T1, T2> {}
+
+/**
+ * @define {string}
+ */
+const GOOD_DEFINE = 'x';
+
+/**
+ * @define {boolean}
+ */
+const BAD_DEFINE = 42;
+
+/**
+ * @define
+ */
+const MISSING_DEFINE = false;
```

Comment:
More like TS_INFERRED_CLOSURE_MISSING_DEFINE

Maybe add also:
```ts
/**
 * @define
 */
const TS_EXPLICIT_CLOSURE_MISSING_DEFINE: boolean = false;
```

Reply:
Changed the names a bit - missing was back from before the type was optional, so it was actually an error.